cmd/mysterium_node/mysterium_node.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"fmt"
```

Comment:
What do you think about start using [goimorts](http://goinbigdata.com/goimports-vs-gofmt/) for formatting a file? It groups imports into 3 parts: 1 - standard library; 2 - external dependencies; 3 - internal dependencies, which makes much easy to read a list of imports.

Reply:
We are using goimports on daily basis ( at least some of us). There is a task to migrate full code base and add checks to CI

Reply:
Fixed imports for my changed files